.devcontainer/devcontainer.json

```diff
@@ -12,12 +12,12 @@
 		"DISPLAY": "${localEnv:DISPLAY}",
 		"NVIDIA_VISIBLE_DEVICES": "all",
 		"NVIDIA_DRIVER_CAPABILITIES": "all",
-		"FASTFASTRTPS_DEFAULT_PROFILES_FILE": "/usr/local/share/middleware_profiles/rtps_udp_profile.xml",
+		"FASTFASTRTPS_DEFAULT_PROFILES_FILE": "/usr/local/share/middleware_profiles/rtps_udp_profile.xml"
 		// "ROS_DOMAIN_ID": "${localEnv:ROS_DOMAIN_ID}"
 	},
 	"initializeCommand": "git config --global --add safe.directory /workspaces/Lunabotics && git reset --soft && git submodule update --init --recursive && git config --global submodule.recurse true && git pull || true",
 	"postCreateCommand": "echo 'source /opt/ros/humble/setup.bash' >> ~/.bashrc && echo 'source /workspaces/Lunabotics/install/setup.bash' >> ~/.bashrc",

```

Comment:
There is still a big issue involving our submodules, specifically isaac ros apriltag I think is breaking things